tools/make.py

```diff
@@ -697,11 +697,9 @@ def version_stamp_pboprefix(module,commitID):
         f.close()
 
         if configtext:
-            patchestext = re.search(r"version.*?=.*?$", configtext, re.DOTALL)
-            if patchestext:
+            if re.search(r"version=(.*?)$", configtext, re.DOTALL):
```

Comment:
Are versions expected to have a specific format? Because this seems like way too broad of a search to me.